pkg/core/interop/crypto/ecdsa.go

```diff
@@ -25,14 +38,26 @@ func ECDSASecp256r1Verify(ic *interop.Context, v *vm.VM) error {
 	if err != nil {
 		return err
 	}
-	res := pkey.Verify(signature, hashToCheck)
+	res := pkey.Verify(curve, signature, hashToCheck)
```

Comment:
Well, it's not correct. `NewPublicKeyFromBytes` has to aware of the curve being used to properly calculate Y coordinate when dealing with compressed key representation. That's why I wrote that "it might be a good reason to switch to PublicKey and PrivateKey from crypto/ecdsa package" in #918.